staging/src/k8s.io/apimachinery/pkg/apis/meta/v1/types.go

```diff
@@ -349,6 +349,13 @@ type ListOptions struct {
 	// add, update, and remove notifications. Specify resourceVersion.
 	// +optional
 	Watch bool `json:"watch,omitempty" protobuf:"varint,3,opt,name=watch"`
+	// AllowBookmarks opts-in for receiving watch events with type Bookmark.
+	// Bookmark events doesn't have any guarantees and it is up-to a server
+	// to decide when such events should be sent.
+	// If Watch field is unset, this field is ignored.
+	// +optional
+	AllowBookmarks bool `json:"allowBookmarks,omitempty" protobuf:"varint,9,opt,name=allowBookmarks"`
```

Comment:
I would prefer `IncludeBookmarks` to match our general `IncludeX` semantics.  While we are allowing bookmarks, we are actually asking for them to be 'included' in the watch.

Reply:
Either that or `SendWatchBookmarks`.

Reply:
I don't care too much-- my reasoning was that "Allow" emphasizes that they are at the server's discretion. "Include" implies that if the client requests them, they will always be sent, which isn't true.

Reply:
"TolerateBookmarks" works for me too-- I was thinking that this is mainly the client expressing that the BOOKMARK type won't make it crash.

Reply:
I think Allow is a better fit in this case.